packages/api/src/api/judge/index.ts

```diff
@@ -0,0 +1,6 @@
+import { Router } from 'express';
+import { post } from './post';
+
+export const judge = Router();
```

Comment:
This isn't being consumed by the parent router so this route isn't being registered